### DISCUSSION TITLE
Best setting JPG B/W in Webp

### DISCUSSION THREAD

**Discussion by ghost** [ImageMagick/ImageMagick#6229] [Help]
### ImageMagick version

7.1.1-6

### Operating system, version and so on

Win x64

### Description

Hello, I made a script to compress my "jpg" "jpeg" "png" into "webp".
[Script Github](https://github.com/ImageMagick/ImageMagick/discussions/6211)

I realized that on some black and white images after compression take up more space.

I have linked one of the images concerned below.
To improve my script, I would like to know if anyone has any ideas to have a good conversion on this type of file in black and white.

Image: [Click Here](https://image.noelshack.com/fichiers/2023/14/3/1680696249-00013.jpg)
or: [Click Here](https://image.noelshack.com/fichiers/2023/14/3/1680698886-00015.jpg)

Thanks, have a good day.

**Comment by urban-warrior** [comment]
We're using ImageMagick 7.1.1.-7 and WEBP (libwebp 1.3.0) returns a smaller sized image as expected.  For 1680696249-00013.jpg we get

```
$ ls -l 1680696249-00013.jpg  1680696249-00013.webp
1082642 1680696249-00013.jpg
 904706 1680696249-00013.webp
```
